src/dso_api/dynamic_api/serializers.py

```diff
@@ -167,29 +189,42 @@ def serializer_factory(model: Type[DynamicModel], flat=None) -> Type[DynamicSeri
 
 def generate_field_serializer(model, model_field, new_attrs, fields, extra_kwargs):
     orig_name = model_field.name
+    # Instead of having to apply camelize() on every response,
+    # create converted field names on the serializer construction.
+    camel_name = snake_to_camel_case(model_field.name)
+    seen = extra_kwargs.get("seen", frozenset())
     if isinstance(model_field, models.ManyToOneRel):
-        # Temporarily skip creation of fields for backwards relations.
+        if not camel_name in seen and camel_name == "buurt":
```

Comment:
Werkt de reverse relation vooralsnog alleen voor buurten, of is dit een special case waar iets bijzonders mee aan de hand is?

Reply:
Dat is een oude commit . Ik had het misschien eerst moeten squashen